frame/utility/src/lib.rs

```diff
@@ -113,8 +113,12 @@ pub mod pallet {
 		BatchInterrupted { index: u32, error: DispatchError },
 		/// Batch of dispatches completed fully with no error.
 		BatchCompleted,
+		/// Batch of dispatches complete but has errors. Index of failing dispatches.
+		BatchCompletedWithErrors { indexes: Vec<u32> },
 		/// A single item within a Batch of dispatches has completed with no error.
 		ItemCompleted,
+		/// A single item within a Batch of dispatches has completed with error.
+		ItemFailed { index: u32, error: DispatchError },
```

Comment:
Sorry if my questions are dumb, this is my first time seeing this pallet:  
Why do you have the index in here? The calls are already ordered.

Reply:
Good catch!

`index` is useless here, I should remove it.

`ItemFailed` originally from `BatchInterrupted { index: u32, error: DispatchError }`, I think I did't consider too much for the `index` field at that time.

Reply:
`index` has been removed, thank you!